kuma/wiki/jinja2/wiki/base.html

```diff
@@ -19,6 +19,11 @@
 
 {% endblock %}
 
+{% block extrahead %}
+    <link rel="dns-prefetch" href="https://interactive-examples.mdn.mozilla.net" pr="0.75" />
+    <link rel="preconnect" href="https://interactive-examples.mdn.mozilla.net" pr="0.75" />
+{% endblock %}
```

Comment:
I felt confident that this is the correct file but, now I am not so sure. When I look at a page such as `http://127.0.0.1:8000/en-US/docs/Experiment:InteractiveEditor/Array.prototype.reduce()` I do not see the `link` elements in the header :(

Reply:
That's because it is overridden in ``document.html``, which is a better place for this code:

https://github.com/mozilla/kuma/blob/a608faaa7c190b094fe15eda33877050e34ecb95/kuma/wiki/jinja2/wiki/document.html#L66-L86